.gitignore

```diff
@@ -0,0 +1,4 @@
+.venv/
+__pycache__/
+*.pyc
+*.sqlite3
```

Comment:
i'm not sure we want to add these to .gitignore.  sqlite should be removed so we can pull updated db changes.  i'm not sure about the .pyc compiled files.  they may be necessary to process changes.

Reply:
.pyc files are only created during execution and should be on the gitignore. The sqlite can be removed. I put it on out of habit from my previous class as it can be seen as a security issue to have db info in github but that should not be an issue for our project